archive/common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/RandomThings.scala

```diff
@@ -23,6 +23,14 @@ trait RandomThings {
     Random.alphanumeric take length mkString
   }
 
+  def randomAlphanumericWithSpaces(length: Int = 8) = {
+    val str = randomAlphanumeric(length).toCharArray
+
+    val spaceIndex = Random.nextInt(str.length - 2) + 1
```

Comment:
I guess the `-2) +1` is to not add spaces at the end or beginning?

Reply:
Yes, I'll add a comment explaining